lib/hawk/http/instrumentation.rb

```diff
@@ -33,7 +33,7 @@ def instrument(type, payload, &block)
               url << '?' << payload[:params].inject('') {|s, (k,v)| s << [k, '=', v, '&'].join }.chomp('&')
             end
 
-            $stderr.printf ">> \033[1mHawk #{type}: #{payload[:method]} #{url} (%.2fms), cache %s\033[0m\n" % [
+            $stderr.printf ">> \033[1mHawk #{type}: #{payload[:method]} #{CGI::unescape(url)} (%.2fms), cache %s\033[0m\n" % [
```

Comment:
Though this would be better written as

```ruby

$stderr.printf ">> \033[1mHawk #{type}: #{payload[:method]} %s (%.2fms), cache %s\033[0m\n" % [
  CGI::unescape(url),
  elapsed,
  payload[:cached] ? 'HIT' : 'MISS'
]
```